tagger.py

```diff
@@ -4,7 +4,13 @@
 import sys
 
 sys.path.insert(0, '.')
+
 from picard.tagger import main
 
-localedir = os.path.join(os.path.dirname(sys.argv[0]), 'locale')
-main(localedir, True)
+# This is needed to find resources when using pyinstaller
+if getattr(sys, 'frozen', False):
+    basedir = sys._MEIPASS
+else:
+    basedir = os.path.dirname(os.path.abspath(__file__))
```

Comment:
Not sure if the change from `os.path.dirname(sys.argv[0])` to `os.path.dirname(os.path.abspath(__file__))` is strictly needed to support pyinstaller.
I don't know it is better or worse, but this branch isn't used by pyinstaller, right ?
 Is there a specific reason ?

Reply:
It's not needed to support pyinstaller, but not sure how sys.argv[0] is a better path for basedir?
sys.argv[0] in this case will be the path to tagger.py if we run it via python tagger.py , which is the same as what the above does.